wemake_python_styleguide/violations/best_practices.py

```diff
@@ -1813,9 +1807,9 @@ class ApproximateConstantViolation(ASTViolation):
 
     Reasoning:
         Some constants are already defined.
-        No need to write them again, use existing values.
-        We just compare numbers as strings and raise this violation
-        when they start with the same chars.
+        No need to write to them again, use existing values.
```

Comment:
I think without "to" is preferable. it's referring to the value being written (rather than the variable name).